Plotting/individual_plots.py

```diff
@@ -70,24 +70,7 @@ def indiv_derived_props_plots(fitspath, restrictMTO = False, revised = False, er
     bin_nondetect = np.where(bin_detect_col == 0.5)[0]
     
     #Define detection and non-detection (with reliable 5007) arrays for individual galaxies  
-    if hbeta_bin == True:
-        indiv_detect = np.where((indiv_bin_detect == 1.0) & (np.isfinite(indiv_OIII5007) == True) & 
-                                (indiv_OIII5007 >= 1e-18) & (indiv_OIII5007 <= 1e-15) & (np.isfinite(indiv_OII) == True) & 
-                                (indiv_OII >= 1e-18) & (indiv_OII <= 1e-15) & (np.isfinite(indiv_HBETA) == True) & 
-                                (indiv_HBETA >= 1e-18) & (indiv_HBETA <= 1e-15) & (indiv_logLHb > 0))[0]
-        indiv_nondetect = np.where((indiv_bin_detect == 0.5) & (np.isfinite(indiv_OIII5007) == True) & 
-                                   (indiv_OIII5007 >= 1e-18) & (indiv_OIII5007 <= 1e-15) & (np.isfinite(indiv_OII) == True) & 
-                                   (indiv_OII >= 1e-18) & (indiv_OII <= 1e-15) & (np.isfinite(indiv_HBETA) == True) & 
-                                   (indiv_HBETA >= 1e-18) & (indiv_HBETA <= 1e-15) & (indiv_logLHb > 0))[0]
-    else:
-        indiv_detect = np.where((indiv_bin_detect == 1.0) & (np.isfinite(indiv_OIII5007) == True) &
-                                (indiv_OIII5007 >= 1e-18) & (indiv_OIII5007 <= 1e-15) & (np.isfinite(indiv_OII) == True) & 
-                                (indiv_OII>= 1e-18) & (indiv_OII<= 1e-15) & (np.isfinite(indiv_HBETA) == True) & 
-                                (indiv_HBETA >= 1e-18) & (indiv_HBETA <= 1e-15))[0]
-        indiv_nondetect = np.where((indiv_bin_detect == 0.5) & (np.isfinite(indiv_OIII5007) == True) & 
-                                   (indiv_OIII5007 >= 1e-18) & (indiv_OIII5007 <= 1e-15) & (np.isfinite(indiv_OII) == True) & 
-                                   (indiv_OII>= 1e-18) & (indiv_OII<= 1e-15) & (np.isfinite(indiv_HBETA) == True) & 
-                                   (indiv_HBETA >= 1e-18) & (indiv_HBETA <= 1e-15))[0]
+    indiv_detect, indiv_nondetect = get_indiv_detect(indiv_OIII5007, indiv_OII, indiv_HBETA, indiv_bin_detect, indiv_logLHb, LHbeta_bins = hbeta_bin)
```

Comment:
Much cleaner with the refactoring

Reply:
You might also consider passing in a `dict()` with the line ratios instead of individual variables.  A cleaner approach would be passing in the `indiv_props_tbl` table, minimizing the need to define each variable.